apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -279,11 +328,11 @@ export function StudentAssessmentInstance({
                                         resLocals.assessment_instance.open &&
                                         instance_question.open,
                                       currentWeight:
-                                        instance_question.points_list_original[
+                                        (instance_question.points_list_original as number[])[
                                           instance_question.number_attempts
-                                        ] - instance_question.max_manual_points,
-                                      pointsList: instance_question.points_list.map(
-                                        (p) => p - instance_question.max_manual_points,
+                                        ] - (instance_question.max_manual_points ?? 0),
+                                      pointsList: (instance_question.points_list as number[]).map(
```

Comment:
```suggestion
                                      pointsList: instance_question.points_list?.map(
```
Nit. The component accepts undefined, so this should be fine. Something similar could be done for the cast for points_list_original.

Reply:
How do the changes from 5260fb8 look?

Reply:
TBD, I'll touch base with them to check. Thanks for your reviews here!